dominion/base.py

```diff
@@ -18,7 +18,11 @@
 
 from images.models import Image
 
```

Comment:
Unnecessary line splitting.

Reply:
The images package is a third-party relatively Dominion, so the blank line is necessary here.

Reply:
Then let's add the blank line [here](https://github.com/tolstoyevsky/dominion/pull/25/files#diff-715432cad783bf48cb9358d0aa4c16c04eb3b203edeb08a515a6163128d7d8f7R28).

Reply:
It's done it https://github.com/tolstoyevsky/dominion/pull/26